editor/editor_file_system.cpp

```diff
@@ -674,6 +675,29 @@ bool EditorFileSystem::_scan_import_support(const Vector<String> &reimports) {
 bool EditorFileSystem::_update_scan_actions() {
 	sources_changed.clear();
 
+	if (first_scan) {
+		// Reimport project theme / base font and load proper project theme first.
+		String project_theme_path = GLOBAL_GET("gui/theme/custom");
+		String project_font_path = GLOBAL_GET("gui/theme/custom_font");
+		Vector<String> reimports;
+
+		if (!project_font_path.is_empty()) {
```

Comment:
Do you think it could be interesting to check if the path really exists on disk before trying to reload it? Usually, the Editor would not try to load a resource that not not exists. Same thing for the `project_theme_path`.